### PR TITLE
Release notes draft

### PR DESCRIPTION
**Pull request by Amruta-Ranade**
Release notes draft for March 26, 2018

### CODE REVIEW

**Comment by jseldess** [comment]
Merging now to get 1.1.7 out. I'll open a follow-up PR with intro text for the 2.0 beta from @nstewart.

**Comment by andreimatei** [comment]
LGTM

File: releases/v2.0-beta.20180326.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+- Added the `cloudstorage.timeout` [cluster setting](../v2.0/cluster-settings.html) for import/export operations. [#23776][#23776]
+
+### SQL Language Changes
+
+- SQL features introduced in CockroachDB v2.0 cannot be used in clusters that are not [upgraded fully to v2.0](../v2.0/upgrade-cockroach-version.html). [#24013][#24013]
+- Added an `escape` option to the `encode()` and `decode()` [built-in functions](../v2.0/functions-and-operators.html). [#23781][#23781]
+- Introduced a series of PostgreSQL-compatible, privilege-related [built-in functions](../v2.0/functions-and-operators.html). [#23839][#23839]
+- Added the `pg_language` table to the `pg_catalog` virtual schema. [#23839][#23839]
+- Added the `anyarray` type to the `pg_type` catalog. [#23836][#23836]
```
Comment (knz) [inline releases/v2.0-beta.20180326.md]: pg_type is a virtual schema, not a catalog.

File: releases/v2.0-beta.20180326.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+- [Retryable errors](../v2.0/transactions.html#error-handling) on schema change operations are now less likely to be returned to clients; more operations are retried internally. [#24050][#24050]
+
+### Command-Line Changes
+
+- Client commands now report a warning if the connection URL is specified by the `--url` as well as some other command-line argument. If you use the `--url` flag, do not set any other connection flags.
```
Comment (knz) [inline releases/v2.0-beta.20180326.md]: This misses the PR link.

Comment (knz) [reply releases/v2.0-beta.20180326.md]: Also the second sentence is not accurate ("If you use the `--url` flag, do not set any other connection flags.") - other flags can "fill in" pieces missing from the URL.

File: releases/v2.0-beta.20180326.md (hunk 1 above)
Comment (jseldess) [reply releases/v2.0-beta.20180326.md]: Thanks, @knz. Updated in https://github.com/cockroachdb/docs/pull/2813.

File: releases/v2.0-beta.20180326.md (hunk 2 above)
Comment (jseldess) [reply releases/v2.0-beta.20180326.md]: I don't know how to get the PR link at this point, but I've updated the text in https://github.com/cockroachdb/docs/pull/2813.

**Pull request merged**